src/main/java/com/ldtteam/structurize/tileentities/TileEntityMultiBlock.java

```diff
@@ -28,7 +31,7 @@
 /**
  * This Class is about the MultiBlock TileEntity which takes care of pushing others around (In a non mean way).
  */
-public class TileEntityMultiBlock extends TileEntity implements ITickableTileEntity
+public class TileEntityMultiBlock extends TileEntity implements ITickableTileEntity, IBlueprintDataProvider
```

Comment:
keeping this on purpose?

Reply:
actually unsure about that yea, I guess it could be useful for testing as structurize doesnt have one atm